scripts/install_platform_with_component_manager.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+set -eo pipefail
+
+CURRENT_DIR=$(cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd)
+source "${CURRENT_DIR}/formats.sh"
+
+if [ -z "${SC_MACHINE_PATH}" ];
+then
+  source "${CURRENT_DIR}/set_vars.sh"
+fi
+
+stage "Install ostis-web-platform"
+
+"${PLATFORM_PATH}/scripts/install_sc_machine_submodule.sh"
+"${PLATFORM_PATH}/scripts/install_sc_machine_dependencies.sh" --dev
+"${PLATFORM_PATH}/scripts/install_sc_component_manager_submodule.sh"
+"${PLATFORM_PATH}/scripts/build_sc_machine.sh"
+"${PLATFORM_PATH}/scripts/build_sc_component_manager.sh"
+"${PLATFORM_PATH}/scripts/build_kb.sh"
+
+stage "OSTIS-web-platform installed successfully"
```

Comment:
Please, add ci for check this script

Reply:
ok, I will add it after merging https://github.com/ostis-ai/sc-component-manager/pull/67

Reply:
Let's merge this PR after marging PR https://github.com/ostis-ai/sc-component-manager/pull/67